packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -404,7 +404,7 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           maxBuy,
           { trySettle, exitAfterBuy = false },
         ) {
-          trace('accept scaledBid offer');
+          trace('accept scaledBid offer', this.state.collateralBrand);
```

Comment:
maybe worth having a `trace` or `log` method on the helper facet

Reply:
didn't seem worth it to me.